lansa-vmss-windows-autoscale-sql-database/azuredeploy.parameters.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+   "parameters": {
+      "msiUri": {
+         "value": "https://lansalpcmsdn.blob.core.windows.net/app/test/AWAMAPP_v14.1.2_en-us.msi"
+      },
+      "vmSku": {
+         "value": "Standard_A2"
+      },
+      "imageOffer": {
+         "value": "lansa-scalable-license"
+      },
+      "vmssName": {
+         "value": "vmsslansa"
```

Comment:
Will do. And whilst we are on the subject of these placeholders, when I execute the script which is in the PR to create the stack, I need to manually replace these placeholders. With GEN-PASSWORD, that is the literal which is passed to the stack. Not very useful. Is there someway to have that script prompt for the replacements?